frontend/src/pages/Assistants/Assistants.styling.tsx

```diff
@@ -0,0 +1,113 @@
+import styled from '@emotion/styled';
+
+const cardHeight = 216;
+
+const AssistantItem = styled('div')`
+    max-width: ${1136 * 0.9}px;
+    height: ${cardHeight}px;
+    width: 100%;
+    border-radius: 0px;
+    padding-left: 1px;
+    margin-top: 24px;
+    display: flex;
+    text-align: left;
+    border: 0.5px solid black;height: ${cardHeight}px;
+`;
+

```

Comment:
Is it possible to use rem and not px here? I think Ronald said something about that this morning. Double check with him.

Reply:
I shall check with him, good catch 😁